### PR TITLE
Prepare the PIT for 2021B

### PR DESCRIPTION
**Pull request by cquiroz**
Signed-off-by: Carlos Quiroz <cquiroz@gemini.edu>

### CODE REVIEW

File: bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/Semester.scala [hunk 1]
```diff
@@ -49,7 +49,7 @@ object Semester {
 
 }
 
-case class Semester(year: Int, half: SemesterOption) {
+sealed case class Semester(year: Int, half: SemesterOption) {
```
Comment (swalker2m) [inline bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/Semester.scala:52]: Did you mean `final`?

Comment (cquiroz) [reply bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/Semester.scala:52]: I guess it should be both. we have defined 2 instances A/B

Comment (cquiroz) [reply bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/Semester.scala]: I got confused with `SemesterOption`, yes it should be `final`

**Review by swalker2m — APPROVED** [review]
:+1:

**Pull request merged**